TrailInsurance/NewClaimViewController+Recording.swift

```diff
@@ -20,6 +20,7 @@ extension NewClaimViewController: AVAudioRecorderDelegate, AVAudioPlayerDelegate
 	}
 
 	func initAVRecordingExtension() {
+//		var rs: AVAudioSession!
```

Comment:
Can we remove this one?

Reply:
It's been removed in the new error handling branch you'll see a PR for later today.